### PR TITLE
fix(web-components): change classification banner to stop overlaying of footer

### PR DESCRIPTION
**Pull request by j212122**
## Summary of the changes
changes banner classification css from fixed to relative to prevent banner overlaying footer

## Related issue

https://github.com/mi6/ic-ui-kit/issues/543

## Checklist
- [x] I have added relevant unit and visual regression tests.
- [ ] I have manually accessibility tested any changes, if relevant.
- [ ] I have ensured any changes match the Figma component library.

### CODE REVIEW

File: packages/web-components/.loki/reference/chrome_iphone7_Web_Components_Footer_With_classification_banner.png
Comment (GCHQ-Developer-112) [inline packages/web-components/.loki/reference/chrome_iphone7_Web_Components_Footer_With_classification_banner.png:1]: The classification banner doesn't appear as expected. It should appear at the bottom of the page here

**Review by GCHQ-Developer-112 — CHANGES_REQUESTED** [review]
Thank you for your contribution ✨ 

The classification banner looks out of place in two of the visual tests. I've left a comment on these two images

Please could you also amend your commit message so it's more reflective of the work you've done

File: packages/web-components/.loki/reference/chrome_laptop_Web_Components_Side_navigation_Default.png
Comment (GCHQ-Developer-530) [inline packages/web-components/.loki/reference/chrome_laptop_Web_Components_Side_navigation_Default.png:1]: The classification banner doesn't look as expected in this visual test, it should overlay the side navigation as well.

**Comment by ad9242** [comment]
Rather than changing the classification banner component, it might be worth having a look at the footer component. As that should already have a `hasClassificationBanner()` method, that should cater for this scenario. Looks like that may be failing for some reason. It doesn't seem to be adding the `classification-spacing` class onto the copyright element.

**Comment by ad9242** [comment]
This doesn't seem to be an issue in the storybook examples  - it works fine here: https://mi6.github.io/ic-ui-kit/branches/develop/?path=/story/web-components-side-navigation--default-story

So appears to be an issue on our site. I still suspect the footer is not detecting that there is a classification banner. It might still require a fix in the component code though

**Comment by gd2910** [comment]
The issues with classification banner on the design-system stem from the component being used in the wrong place. Closing this PR and will create a new branch to scope the design-system to remove this and fix the issue.

**Pull request closed without merging**